### DISCUSSION TITLE
FAQ section on Epiverse's website

### DISCUSSION THREAD

**Discussion by CarmenTamayo** [epiverse-trace/epiverse-trace.github.io#152] [General]
There are some recurring questions about the project that we are asked during events or while talking to external partners, and it would be useful to have resource that we can point people towards, such as questions about the development plans (e.g., is there a plan to include genomic data? will spatial models be included in {epidemics}, or in a separate package?).
This would also be useful to answer questions in a uniform way, to avoid people receiving different information depending on who they ask in the project.
We could all add questions and answers as we find necessary when interacting with external people, e.g., during showcases.

**Comment by chartgerink** [comment]
@CarmenTamayo I am happy to add a FAQ section if you have a list of questions + answers that should go in. I think this is a great addition for the website overall.

Maybe we can start by collecting questions that people received here so we can identify which ones are more frequent?
